app/assets/stylesheets/components/navbar.scss

```diff
@@ -58,8 +58,11 @@
       background-position: center center;
       background-size: cover;
     }
-    .drop-down-link:hover {
-      color: $primary;
+    .drop-down-link{
+      color: grey;
```

Comment:
this should be a variable instead of a color I think.... (looks like you forgot the $